buildsystem/CompileSuite/autoTests/new_commits.sh

```diff
@@ -94,7 +94,7 @@ touch "$thisDir"runGuard
             #export PIC_COMPILE_SUITE_CMAKE="-DPIC_ENABLE_PNG=OFF -DALPAKA_CUDA_ARCH=35"
             export PIC_BACKEND="cuda"
             . /etc/profile
-            module load gcc/4.9.4 boost/1.62.0 cmake/3.7.0 cuda/8.0.44 openmpi/1.10.4
+            module load gcc/4.9.4 boost/1.62.0 cmake/3.10.0 cuda/8.0.44 openmpi/1.10.4
```

Comment:
QUESTION:
Why does the compile suite uses cmake/3.10.0 while all other use cmake 3.10.1?

Reply:
the compile suite tests our *minimal* software requirements, so we know they still work.

individual systems such as hypnos might use newer versions